CODEOWNERS

```
@@ -0,0 +1,10 @@
+# https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners
+# https://github.blog/2017-07-06-introducing-code-owners/
+
+# Assign code owners that will automatically get asked to review Pull Requests
+# The last matching pattern takes the most precedence.
+
+# These owners will be the default owners for everything in the repo.
+# Unless a later match takes precedence, they will be requested for
+# review when someone opens a pull request.
+* @EGI-Federation/admins @EGI-Federation/operations
```

```suggestion
* @EGI-Federation/admins @EGI-Federation/sdis 
```

Same as the other repo